### PR TITLE
update export drivers to generate sys

### PR DESCRIPTION
**Pull request by tedhudek**

### CODE REVIEW

File: windows-driver-docs-pr/kernel/creating-export-drivers.md [hunk 1]
```diff
@@ -26,9 +26,10 @@ Standard drivers can also function as export drivers. For a driver to function i
 To create an export driver in Visual Studio, use the following procedure:
 
 1. Create a new project from a template, such as **Empty WDM Driver**.
-2. In project settings, set **General -> Configuration Type** to **Dynamic Library (.dll)**.
-3. Set **Linker -> Advanced -> No Entry Point** to **Yes (/NOENTRY)**.
-4. Add a module definition file to the project, for example:
+1. In project settings, set **General -> Configuration Type** to **Dynamic Library (.dll)**.
```
Comment (hongyver) [inline windows-driver-docs-pr/kernel/creating-export-drivers.md:29]: delete

File: windows-driver-docs-pr/kernel/creating-export-drivers.md [hunk 2]
```diff
@@ -26,9 +26,10 @@ Standard drivers can also function as export drivers. For a driver to function i
...
+1. Set **Linker -> Advanced -> No Entry Point** to **Yes (/NOENTRY)**.
```
Comment (hongyver) [inline windows-driver-docs-pr/kernel/creating-export-drivers.md:30]: delete

File: windows-driver-docs-pr/kernel/creating-export-drivers.md [hunk 3]
```diff
@@ -26,9 +26,10 @@ Standard drivers can also function as export drivers. For a driver to function i
...
+1. Add main.cpp.
```
Comment (hongyver) [inline windows-driver-docs-pr/kernel/creating-export-drivers.md:31]: delete

File: windows-driver-docs-pr/kernel/creating-export-drivers.md [hunk 4]
```diff
@@ -26,9 +26,10 @@ Standard drivers can also function as export drivers. For a driver to function i
...
+1. Add a module definition (.def) file to the project, for example:
 
   ```
   LIBRARY mydriver
```
Comment (hongyver) [inline windows-driver-docs-pr/kernel/creating-export-drivers.md:35]: LIBRARY mydriver.sys

**Comment by tedhudek** [comment]
Fixed in #3428

**Pull request closed without merging**